ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/streaming/QueryStreamWriter.java

```diff
@@ -120,7 +124,7 @@ public void write(OutputStream out) throws IOException {
   }
 
   private void write(OutputStream output, GenericRow row) throws IOException {
-    objectMapper.writeValue(output, new StreamedRow(row));
+    output.write(jsonSerializer.serialize("", row));
```

Comment:
Why? This breaks the work done in https://github.com/confluentinc/ksql/pull/1337.

Reply:
Using connect classes for JSON serialization. This was done before #1337. I'll make sure it won't break it when rebasing this PR.